projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.scss

```diff
@@ -60,3 +60,16 @@ img {
   -o-transform-origin: left top;
   transform-origin: left top;
 }
+
+.dragScroll {
```

Comment:
is it possible to add css to a directive?

Reply:
nope but i think i can just add it to the media viewer and as it doesnt get override it will work, i believe